cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -78,6 +78,7 @@ spec:
                 provider:
                   type: string
                 selectors: {}
+              nullable: true
```

Comment:
Is there a different setting than `nullable: true`? According to [this comment](https://github.com/rook/rook/issues/3660#issuecomment-590347497), we can't use it until we drop support for older than k8s 1.14. 

But maybe we could add the `nullable` settings to the schema and still keep supporting those versions:
- The integration tests could create the schema without `nullable`, to still run against older versions
- The docs could mention that the common.yaml would need to be changed to remove `nullable`

This would be a softer way to stop supporting versions older than 1.14. The pain level would increase, but it would still be possible...

Reply:
wondering. I just copied form https://github.com/rook/rook/pull/5862/files#diff-ee4f8509d6cbc023717cf822fd7c1ec5R144  

hm.